tests/postgres_tests/test_aggregates.py

```diff
@@ -0,0 +1,252 @@
+import unittest
+
+from django.contrib.postgres.aggregates import (
+    ArrayAgg, BitAnd, BitOr, BoolAnd, BoolOr, Corr, CovarPop, RegrAvgX,
+    RegrAvgY, RegrCount, RegrIntercept, RegrR2, RegrSlope, RegrSXX, RegrSXY,
+    RegrSYY, StatFunc, StringAgg,
+)
+from django.db import connection
+from django.db.models.expressions import F, Value
+from django.test import TestCase
+from django.test.utils import Approximate
+
+from .models import GeneralTestModel, StatTestModel
+
+
+@unittest.skipUnless(connection.vendor == 'postgresql', 'PostgreSQL required')
+class TestGeneralAggregate(TestCase):
+    fixtures = ["aggregation_general.json"]
```

Comment:
I think we should avoid writing new test suites that use fixtures. Fixture loading is extremely slow, and it's actually harder, IMO, to follow what the data should look like once you've aggregated it.

I would suggest either creating all the data in a setUp method, or creating the data you need at the top of each test.